openslides/users/models.py

```diff
@@ -3,12 +3,13 @@
 from random import choice
 
 from django.contrib.auth.hashers import make_password
-from django.contrib.auth.models import (
+from django.contrib.auth.models import (  # noqa
```

Comment:
Here I would prefere separating the imports that are here just for other files to import them and the really required imports. But maybe isort doesn't like this ...